user_guide/src/howcani/df/row_col_selection.md

```diff
@@ -0,0 +1,23 @@
+# Row and column selection
+
+Selection of rows and columns is similar to other DataFrame libraries.
+
+## Column selection
+
+```python
+df[['a','b']]
+```
+
+```text
+{{#include ../../outputs/row_col_selection/col_selection.txt}}
+```
+
+## Row selection
+
+```python
+df[0:2]
+```
+
+```text
+{{#include ../../outputs/row_col_selection/row_selection.txt}}
+```
```

Comment:
Could add some more complicate slicing example: check the [`__getitem__`](https://github.com/pola-rs/polars/blob/master/py-polars/polars/frame.py#L473) code.

Reply:
Next iteration?

Reply:
@ritchie46 now or later?

Reply:
Later is fine. Then this is alreay in the guide. :+1: